src/libint_3center.F

```diff
@@ -4,14 +4,15 @@
 !--------------------------------------------------------------------------------------------------!
 
 ! **************************************************************************************************
-!> \brief 3-center electron repulsion integral routines based on libint2
+!> \brief 2- and 3-center electron repulsion integral routines based on libint2
```

Comment:
Maybe we might want to change the name of this module if it takes care of both 2-center and 3-center

Reply:
Yes, I renamed it to `libint_2c_3c` (as well as a couple of other module names that have been generalized).